editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -58,11 +77,50 @@ function MapAndLabelComponent(props: Props) {
                 name="fn"
                 placeholder={"Data Field"}
                 value={formik.values.fn}
+                onChange={formik.handleChange}
                 required
               />
             </InputRow>
           </InputGroup>
         </ModalSectionContent>
+        <ModalSectionContent title="Map Formatting">
+          <InputGroup>
+            <InputRow>
+              <InputRowItem>
+                <ColorPicker
+                  label="Drawing Colour"
+                  color={formik.values.drawColour}
+                  onChange={(color) => {
+                    formik.setFieldValue("drawColour", color);
```

Comment:
The mix of `ou` & `o` here feels easy to trip over in the future - we should probably pick one way for consistency?

Reply:
Feels like the American spelling is most used in the map component so we should move forward with that?

Reply:
Yep I think aligning editor to map makes sense here, and we have majority `color` in planx-new too because dependencies like MUI etc !

Reply:
For this, it looks like user facing text uses ``Colour`` and variable names and text in the code base uses ``color`` so I'll follow that pattern and switch my variable names to ``color`` but keep display text as ``colour``